index.rst

```diff
@@ -6,24 +6,498 @@
 
 .. TODO: Delete the note below before merging new content to the main branch.
 
-.. note::
-
-   **This technote is a work-in-progress.**
+.. Make in-text citations with: :cite:`bibkey`.
+.. Uncomment to use citations
+.. .. rubric:: References
+..
+.. .. bibliography:: local.bib lsstbib/books.bib lsstbib/lsst.bib lsstbib/lsst-dm.bib lsstbib/refs.bib lsstbib/refs_ads.bib
+..    :style: lsst_aa
 
 Abstract
 ========
 
-Principles, guidelines, conventions, formatting style, and workflow for tutorials developed by the Community Engagement Team (CET).
+This document provides guidelines for anyone creating or updating tutorials for the Vera C. Rubin Observatory Community Engagement Team (CET) and the Rubin Observatory science community.
+This includes all community-facing demonstrations of how to use the Rubin Science Platform (RSP) or how to analyze Legacy Survey of Space and Time (LSST) data.
 
-Add content here
-================
+Pedagogical principles
+======================
 
-Add content here.
-See the `reStructuredText Style Guide <https://developer.lsst.io/restructuredtext/style.html>`__ to learn how to create sections, links, images, tables, equations, and more.
+All tutorial developers should adhere to the following principles.
 
-.. Make in-text citations with: :cite:`bibkey`.
-.. Uncomment to use citations
-.. .. rubric:: References
-.. 
-.. .. bibliography:: local.bib lsstbib/books.bib lsstbib/lsst.bib lsstbib/lsst-dm.bib lsstbib/refs.bib lsstbib/refs_ads.bib
-..    :style: lsst_aa
+Inclusivity
+-----------
+
+Offensive or exclusionary language is never permitted (e.g., violent or ableist terms).
```

Comment:
This is rather terse - is there a code of conduct or something similar that we could link to? (Either a DP0-specific statement, or the "standard" Rubin or NOIRLab statements?)